lib/test/python2/sqlcipher.py

```diff
@@ -97,7 +97,11 @@ def assertDatabaseError(self, password):
             col_value = self.queryData(conn)
             self.assertIsNone(col_value)
         except sqlite.DatabaseError as ex:
-            self.assertEqual('file is encrypted or is not a database', str(ex))
+            if sqlite.libsqlcipher_version < (3, 20, 1):
+                # https://github.com/sqlcipher/sqlcipher/commit/821782a23b06ab2fd774ae2500d18e1cc2a82a80
+                self.assertEqual('file is encrypted or is not a database', str(ex))
```

Comment:
No need to really check the version. Because I don't think the test really cares about the exact message but rather the substance behind the message. Either just check for either string or even simplifying the assert string would probably be fine to just "is not a database"